src/trio/_tests/test_ssl.py

```diff
@@ -251,9 +256,10 @@ def __init__(self, sleeper: None = None) -> None:
             "simultaneous calls to PyOpenSSLEchoStream.receive_some",
         )
 
+        self.sleeper: Callable[[str], Awaitable[None]]
         if sleeper is None:
 
-            async def no_op_sleeper(_: object) -> None:
+            async def no_op_sleeper(_: str) -> None:
                 return
 
```

Comment:
nit: there's really no reason to narrow this.

Reply:
Reverted in c4fccd0